<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NumberButtons</class>
 <widget class="QWidget" name="NumberButtons">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>215</width>
    <height>260</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget{
background-color: rgb(51,153,255);
}

QPushButton{
outline:none;
background-color: rgb(51,153,255);
border:3px solid white;
font:23pt  &quot;SimHei&quot;;
color:white;
}

/*QPushButton:disabled
{
background-color: rgb(51,153,255);
border:3px solid rgb(217, 217, 217);
font:23pt, &quot;SimHei&quot;;
color: rgb(217, 217, 217);
}*/</string>
  </property>
  <widget class="QPushButton" name="btn5">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>55</y>
     <width>65</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn3">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>5</y>
     <width>65</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn1">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>5</y>
     <width>65</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">outline:none;
background-color: rgb(51,153,255);
border:3px solid white;
font:30pt  &quot;SimHei&quot;;
color:white;</string>
   </property>
   <property name="text">
    <string>1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn4">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>55</y>
     <width>65</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn9">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>105</y>
     <width>65</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnClear">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>155</y>
     <width>135</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>清除</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn0">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>155</y>
     <width>65</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn7">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>105</y>
     <width>65</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn6">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>55</y>
     <width>65</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn2">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>5</y>
     <width>65</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn8">
   <property name="geometry">
    <rect>
     <x>75</x>
     <y>105</y>
     <width>65</width>
     <height>45</height>
    </rect>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnConfirm">
   <property name="geometry">
    <rect>
     <x>5</x>
     <y>205</y>
     <width>65</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>确认</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBack">
   <property name="geometry">
    <rect>
     <x>145</x>
     <y>205</y>
     <width>65</width>
     <height>45</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
